src/components/TokenBalance.test.tsx

```diff
@@ -57,11 +57,7 @@ jest.mocked(getDynamicConfigParams).mockReturnValue({
 describe('FiatExchangeTokenBalance and HomeTokenBalance', () => {
   beforeEach(() => {
     jest.clearAllMocks()
-    jest
-      .mocked(getFeatureGate)
-      .mockImplementation(
-        (featureGate) => featureGate !== StatsigFeatureGates.SHOW_ASSET_DETAILS_SCREEN
-      )
+    jest.mocked(getFeatureGate).mockImplementation(() => true)
```

Comment:
This is controlling multiple features flags. Open to other options on how to handle this.

Reply:
Fixed in 6fb2c1d1f5fc2f5efbb05fe375e373bfd32718c4!